src/NotebookExperiments.ts

```diff
@@ -100,17 +101,18 @@ export class NotebookExperimentWidget extends Widget {
         let table = this.createResultTable();
 
         // populate the header
-        for(var i=0; i < json.models.length; i++) {
-          var model:JSONObject = json.models[i];
-          var training: JSONObject = <JSONObject> model.training;
-          var training_status: JSONObject = <JSONObject> training.training_status;
-
-          var tableRow = table.insertRow();
-          tableRow.className = "dlw-Table-experiments";
+        for(let i=0; i < json.models.length; i++) {
+          let model:JSONObject = json.models[i];
+          let training: JSONObject = <JSONObject> model.training;
+          let training_status: JSONObject = <JSONObject> training.training_status;
+          let statusText = training_status.status.toString();
+
+          let tableRow = table.insertRow();
+          tableRow.className = statusText === "COMPLETED" ?  "dlw-Table-experiments-completed" : "dlw-Table-experiments";
```

Comment:
I am ok with this approach, but want to raise the question... is it better to change the style of the whole row versus only the cell or text of the status column as in the FfDL UI?

Reply:
It may help readability in case the experiments tab size is narrower, the user wouldn't need to scroll all the way to the status column, since the entire job info would be under the same context. We can confirm the best approach with a designer as well.